contracts/script/D01DeployTracker.s.sol

```diff
@@ -55,6 +58,7 @@ contract Deploy is Script {
             "Nonce Run Club June",
             10,
             1717171200, // June 1
+            1717171200, // June 1
             1719676800, // June 30
             official_verifier,
             0.025 ether
```

Comment:
Can we also make all the "stakes" something like 100 ALI? 
0.00x makes it looks cheap, we probably want to fully replicate the mainnet version we will first release, and I image that would be using USDC, so a reasonable stake might be 50~ 100 USDC